### DISCUSSION TITLE
GitHub Certifications for Community Contributors Discussion

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#121016] [Other Feature Feedback, Questions, & Ideas]
We want want to hear about your experience with GitHub Certifications! 

This is a feedback Discussion for those who were awarded a voucher for a GitHub Certification exam. 

Let us know what you liked, disliked, and any other comments related to Certifications below.

You can find all the information you may need for your Certifications journey on our [site](https://resources.github.com/learn/certifications/) and if you have any issues, don't hesitate to reach out to [gh-certification@github.com](mailto:gh-certification@github.com).

We currently offer four exams for certification: 
- [GitHub Foundations](https://learn.microsoft.com/en-us/collections/o1njfe825p602p)
- [GitHub Actions](https://learn.microsoft.com/en-us/users/githubtraining/collections/n5p4a5z7keznp5)
- [GitHub Advance Security (GHAS)](https://learn.microsoft.com/en-us/users/githubtraining/collections/rqymc6yw8q5rey)
- [GitHub Administration](https://learn.microsoft.com/en-us/users/githubtraining/collections/mom7u1gzjdxw03). 

Thanks for participating!

**Comment by Sitevity** [comment]
Accept my answer 

 I wanna Earn Badge

**Reply by ebndev** [reply]
Thanks so much for posting in our Community @Sitevity. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by charlestanev** [comment]
Hello everyone,

I'm excited to join the discussion about GitHub Certifications for Community Contributors! It's great to see the platform offering certifications to recognize and validate our skills. I'm particularly interested in hearing about your experiences with the exams and any feedback you have. Let's share our thoughts and insights to help each other succeed in our certification journeys.

Looking forward to engaging with all of you!

**Comment by charlestanev** [comment]
I'm new to the community and eager to learn more about GitHub Certifications. It's inspiring to see how GitHub is empowering contributors with opportunities to enhance their skills and credentials. I'm particularly curious about the different certification exams available and the insights shared by those who have taken them. Excited to be part of this discussion and learn from your experiences!

**Comment by rizwanwattoo** [comment]
Great

**Reply by Khoironevi** [reply]
Ya

**Reply by crystalovelson** [reply]
I believe someone is hacking me with this app what do I do

**Comment by rizwanwattoo** [comment]
Good

**Reply by Khoironevi** [reply]
Thanks you

**Reply by Oladigbs18** [reply]
I was also given a discount but I was unable to do the exam because my examiner had to cancel it due to the fact that I wasn't able to provide a means of identification

**Comment by ghost** [comment]
I got blocked from TeamGalacticraft for an unknown reason or an error, and they gave me no details at all. GitHub AI has all kinds of errors. I wish they would just faze out AI powered blocking. I didn't even do anything!?

**Comment by Janadeepa** [comment]
best

**Comment by maanu1234** [comment]
> We want want to hear about your experience with GitHub Certifications!
> 
> This is a feedback Discussion for those who were awarded a voucher for a GitHub Certification exam.
> 
> Let us know what you liked, disliked, and any other comments related to Certifications below.
> 
> You can find all the information you may need for your Certifications journey on our [site](https://resources.github.com/learn/certifications/) and if you have any issues, don't hesitate to reach out to [gh-certification@github.com](mailto:gh-certification@github.com).
> 
> We currently offer four exams for certification:
> 
> * [GitHub Foundations](https://learn.microsoft.com/en-us/collections/o1njfe825p602p)
> * [GitHub Actions](https://learn.microsoft.com/en-us/users/githubtraining/collections/n5p4a5z7keznp5)
> * [GitHub Advance Security (GHAS)](https://learn.microsoft.com/en-us/users/githubtraining/collections/rqymc6yw8q5rey)
> * [GitHub Administration](https://learn.microsoft.com/en-us/users/githubtraining/collections/mom7u1gzjdxw03).
> 
> Thanks for participating!

**Comment by Ibneaminkhan708** [comment]
www.bing.com

**Comment by SUDEEP-M-SHETTY** [comment]
**Microsoft Reactor is a community space where technology professionals can meet, learn, and connect with each other. It provides a platform for various events, workshops, and networking opportunities, focusing on Microsoft products and technologies.**

**Events**: The Reactor hosts various events, including conferences, workshops, and meetups. These events are free and open to everyone, focusing on topics such as AI, cloud computing, and machine learning.

*By attending these events you can redeem coupon code they provide to limited people at the end of the event which can apply for obtaining a voucher for a GitHub Certification exam.*

**Comment by dennysis** [comment]
great but how does one get it

**Comment by davevad93** [comment]
Hi @queenofcorgis ,

I just found out about this discussion and maybe it's a bit late, what are the requirements to get a voucher for GitHub certifications? Is it a time-limited thing or is it an initiative that will continue in the long term?

Thank you in advance.

**Comment by queenofcorgis** [comment]
@davevad93 @dennysis thanks for your interest. This was a program we ran back in April for a select group of community members that were nominated. Unfortunately, we are not distributing any more vouchers at this moment, but we've noted your interest!

**Comment by maheshsehajpal** [comment]
that's great

**Comment by tjmanoj** [comment]
Response #10086
